File: T.T_testing/n3+1.py
num1 = int(input())
num2 = int(input())
maxval = 0

while num1!=num2:
    lopnum1 = num1
    stk = 1
    while lopnum1 != 1:
        if lopnum1 % 2 == 0:
            lopnum1 /= 2
        else:
            lopnum1 = (lopnum1*3)+1
        stk += 1
    if stk > maxval:
        maxval = stk
    num1 += 1
print(maxval)

File: T.T_testing/n3+1.cpp
#include <iostream>
using namespace std;

int main() {
  int a,b,c; 

  int maxval = 0;
  int stk = 1;
  int ans = 0;

  // cout << "Type a number: "; 
  cin >> a; 
  // cout << "Type a number: ";
  cin >> b;
  while(a!= b){
    c = a;
    stk = 1;
    while(c!=1){
                if(c%2==0){
                    c /= 2;
                }
                else{
                    c = c*3+1;
                }
                stk += 1;
                // System.out.println(stk);
            }
            if(stk>ans){
                ans = stk;
            }
            a++;
            // cout << a;
            
  }
  cout << ans;
}